app/Http/Controllers/AdminController.php

```diff
@@ -41,6 +45,24 @@ public function assignValidator($id)
     }
 
     /**
+     * Admin view canceled events
+     */
+    public function viewDeletedEvents()
+    {
+        $events =  Event::onlyTrashed()->get();
+        return view('admin.view-deleted-events', compact('events'));
+    }
+
+    /**
+     * Admin restore canceled event
+     */
+    public function restoreEvent($id)
```

Comment:
We will either keep this function here and bring `destroy` function too or move this function to `EventController`